package.json

```diff
@@ -21,34 +21,24 @@
   "dependencies": {
     "@mdi/font": "3.3.92",
     "@tokend/js-sdk": "0.3.15",
-    "@tokend/wallet.js": "git+https://github.com/tokend/wallet.js.git",
-    "accounting": "0.4.1",
     "bignumber.js": "8.0.1",
     "clipboard": "2.0.4",
     "d3": "5.7.0",
-    "flatpickr": "4.5.2",
     "husky": "1.3.1",
     "i18next": "13.1.0",
-    "libphonenumber-js": "1.7.4",
     "lodash": "4.17.11",
     "moment": "2.23.0",
-    "node-polyglot": "2.3.0",
     "numeral": "2.0.6",
-    "tokend-js-sdk": "git+https://github.com/tokend/js-sdk.git",
-    "ua-parser-js": "0.7.19",
-    "vee-validate": "2.1.5",
     "vue": "2.5.21",
     "vue-flatpickr-component": "8.1.0",
     "vue-markdown": "2.2.4",
-    "vue-material": "1.0.0-beta-7",
     "vue-qr": "1.5.2",
-    "vue-qrcode-component": "2.1.1",
-    "vue-resource": "1.5.1",
```

Comment:
Please, keep `vue-resource` until that TODO is resolved. @max-nes @ValeriaNARDET @quinn71 

https://github.com/tokend/web-client/blob/65402043f10538fe84bab77d4a3df8facc522efa/src/js/helpers/document-uploader.js#L71-L72